mlflow/pmdarima.py

```diff
@@ -138,73 +144,62 @@ def save_model(
-
-                     .. Note:: Experimental: This parameter may change or be removed in a future
-                                             release without warning.
-
-    .. code-block:: python
-        :caption: Example
-
-        import pandas as pd
-        import mlflow
-        import pmdarima
-
-        # Specify locations of source data and the model artifact
-        SOURCE_DATA = "https://raw.githubusercontent.com/facebook/prophet/master/examples/example_retail_sales.csv"
-        ARTIFACT_PATH = "model"
-
-        # Read data and recode columns
-        sales_data = pd.read_csv(SOURCE_DATA)
-        sales_data.rename(columns={"y": "sales", "ds": "date"}, inplace=True)
-
-        # Split the data into train/test
-        train_size = int(0.8 * len(sales_data))
-        train, test = sales_data[:train_size], sales_data[train_size:]
-
-        with mlflow.start_run():
-            # Create the model
-            model = pmdarima.auto_arima(train["sales"], seasonal=True, m=12)
+    Save a pmdarima ``ARIMA`` model or ``Pipeline`` object to a path on the local file system.
 
-            # Save the model to the specified path
-            mlflow.pmdarima.save_model(model, "model")
+    Args:
+        pmdarima_model: pmdarima ``ARIMA`` or ``Pipeline`` model that has been ``fit`` on a
+                        temporal series.
+        path: Local path destination for the serialized model (in pickle format) is to be saved.
+        conda_env: {{ conda_env }}
+        code_paths: A list of local filesystem paths to Python file dependencies (or directories
+                    containing file dependencies). These files are *prepended* to the system
+                    path when the model is loaded.
+        mlflow_model: :py:mod:`mlflow.models.Model` this flavor is being added to.
+        signature: an instance of the :py:class:`ModelSignature <mlflow.models.ModelSignature>`
+                   class that describes the model's inputs and outputs. If not specified but an
+                   ``input_example`` is supplied, a signature will be automatically inferred
+                   based on the supplied input example and model. To disable automatic signature
+                   inference when providing an input example, set ``signature`` to ``False``.
+                   To manually infer a model signature, call
+                   :py:func:`infer_signature() <mlflow.models.infer_signature>` on datasets
+                   with valid model inputs, such as a training dataset with the target column
+                   omitted, and valid model outputs, like model predictions made on the training
+                   dataset.
+        input_example: {{ input_example }}
+        pip_requirements: {{ pip_requirements }}
+        extra_pip_requirements: {{ extra_pip_requirements }}
+        metadata: Custom metadata dictionary passed to the model and stored in the MLmodel file.
+
+    Note:
```

Comment:
`..` prefix dropped from Note

Reply:
can we also replace the vertical alignment to 4-space indent?